ccdproc/combiner.py

```diff
@@ -137,9 +135,10 @@ def weights(self, value):
                 if value.shape == self.data_arr.data.shape:
                     self._weights = value
                 else:
-                    raise ValueError("dimensions of weights do not match data")
+                    raise ValueError(
+                        "dimensions of weights do not match data.")
             else:
-                raise TypeError("mask must be a Numpy array")
+                raise TypeError("mask must be a numpy ndarray.")
```

Comment:
Or [`NumPy`](http://www.numpy.org/)? I've edited it to `numpy.ndarray` that's consistent with the rest of documentation.